projects/storefrontapp-e2e-cypress/cypress/integration/b2b/regression/quick-order/b2b-quick-order.e2e-spec.ts

```diff
@@ -0,0 +1,80 @@
+import * as quickOrder from '../../../../helpers/b2b/b2b-quick-order';
+import { viewportContext } from '../../../../helpers/viewport-context';
+import * as sampleData from '../../../../sample-data/b2b-quick-order';
+
+context('B2B - Quick Order', () => {
+  viewportContext(['mobile', 'desktop'], () => {
+    before(() => {
+      cy.window().then((win) => win.sessionStorage.clear());
+      cy.window().then((win) => win.localStorage.clear());
+      cy.clearLocalStorageMemory();
+    });
+
+    describe('Accessibility - keyboarding', () => {
```

Comment:
I believe not all your test below is checking for accessibility - keyboarding.

GC would like the e2e test case for accessibility to be run at the last of the sequence for the e2e test.

```
describe('Quick order page', () => {})

describe('Cart page', () => {})

describe('Accessibility - keyboarding', () => {})
```